backend/globaleaks/handlers/accreditator/__init__.py

```diff
@@ -0,0 +1,158 @@
+from globaleaks.handlers.accreditator.services import accreditation, get_all_accreditation, persistent_drop, \
```

Comment:
This entire module (accreditor) seems to be implemeted without following considering the existing "Signup" module  and with significative difference with the specifications provided before the development: https://github.com/globaleaks/globaleaks-whistleblowing-software/issues/4107

It seems that the implementation uses a proxy in front of the software as a third party component that is specifically denied by Thread modeling of the application: https://docs.globaleaks.org/en/stable/security/ThreatModel.html#network-and-reverse-proxies

It is unfortunately not possible to merge this work without:
- reimplementing the feature on top of the existing signup module and the specifications provided in https://github.com/globaleaks/globaleaks-whistleblowing-software/issues/4107